vms/platformvm/block/builder/builder.go

```diff
@@ -239,13 +245,20 @@ func (b *builder) BuildBlock(context.Context) (snowman.Block, error) {
 		return nil, fmt.Errorf("could not calculate next staker change time: %w", err)
 	}
 
+	var pChainHeight uint64
```

Comment:
I'm assuming `pChainHeight=0` is a valid input to `buildBlock` and subsequent function calls? Is this documented anywhere? I traced the call as far as [BitSetSignature.Verify](https://github.com/ava-labs/avalanchego/blob/f17ea6a7ab4036c41b693e47b94d8f0c81cb69ec/vms/platformvm/warp/signature.go#L70) and wasn't seeing where this case was handled.